README.md

```diff
@@ -141,6 +141,19 @@ This means you can scale down to zero when idle, which means you can use larger
 
 Metrics are collected with a Lambda function, polling every minute.
 
+## Terminating the instance after job is complete
+
+You may set `BuildkiteTerminateInstanceAfterJob` to `true` to force the instance to terminate after it completes a job. Setting this value to `true` tells the stack to apply the following configurations:
+
+1. Sets `disconnect-after-job` and `disconnect-after-job-timeout` in the `buildkite-agent.cfg` file.
+2. Adds `ExecStopPost` steps to the agent's systemd service to mark the instance as unhealthy and to stop the instance.
+
+While not enforced, it is highly recommended if you also set to set your `AgentsPerInstance` value to `1`.
```

Comment:
@tduffield looks like this needs some cleaning up. Was it supposed to be this?

While not enforced, it is highly recommended you also set the `AgentsPerInstance` value to `1`.

Reply:
Thanks!